curriculum/challenges/english/25-front-end-development/lecture-working-with-tables/672a4e04a24858778f57ebfe.md

```diff
@@ -10,6 +10,113 @@ dashedName: what-are-html-tables-used-for
 
 Watch the video lecture and answer the questions below.
 
+# --transcript--
+
+What are HTML tables used for and what should they not be used for?
+
+HTML tables aren't used as much these days as they used to be but as a frontend developer, you should still be familiar with them.
+
+Tables are one of the earliest ways devs had for displaying data in a browser way back in the 1990s.
+
+Here’s an example of code used to generate a table from the U.S. Bureau of Labor Statistics:
+
+```html
+<table id="quickfacts">
+    <thead>
+        <tr>
+            <th colspan="2">Quick Facts: Software Developers, Quality Assurance Analysts, and Testers</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th>2023 Median Pay</th>
+            <td>
+                $130,160 per year
+                <br>$62.58 per hour
+            </td>
+        </tr>
+        <tr>
+            <th>Typical Entry-Level Education</th>
+            <td>Bachelor's degree</td>
+        </tr>
+        <tr>
+            <th>Work Experience in a Related Occupation</th>
+            <td>None</td>
+        </tr>
+        <tr>
+            <th>On-the-job Training</th>
+            <td>None</td>
+        </tr>
+        <tr>
+            <th>Number of Jobs, 2022</th>
+            <td>1,795,300</td>
+        </tr>
+        <tr>
+            <th>Job Outlook, 2022-32</th>
+            <td>25% (Much faster than average)</td>
+        </tr>
+        <tr>
+            <th>Employment Change, 2022-32</th>
+            <td>451,200</td>
+        </tr>
+    </tbody>
+    <tfoot>
+    	<tr>
+    		<th>If this table had a footer it would be here.</th>
+    	</tr>
+    </tfoot>
+</table>
+```
+
+As you can see, there's a main `<table>` element with an `id` set to `quickfacts`. Inside it, the table has a Table Head element, `<thead>`, Table Body element, `<tbody>`, and a Table Foot element, `<tfoot>`.
+
+The Table Head, Body, and Foot elements can each contain some number of Table Rows, `<tr>`. And each Table Row can contain a Table Header `<th>` which labels the data in that row. It can also contain some number of individual data cells, called Table Data, `<td>`.
```

Comment:
```suggestion
The table head, body, and foot elements can each contain some number of table rows, `<tr>`. And each table row can contain a table header `<th>` which labels the data in that row. It can also contain some number of individual data cells, called table data, `<td>`.
```